Cargo.toml

```diff
@@ -110,7 +110,7 @@ slog-term = "2.4"
 slog_derive = "0.1"
 parking_lot = "0.10"
 sst_importer = { path = "components/sst_importer" }
-sysinfo = { git = "https://github.com/tikv/sysinfo.git", branch = "tikv"}
+sysinfo = { git = "https://github.com/tikv/sysinfo.git", branch = "tikv2"}
```

Comment:
Can we merge `tikv2` into `tikv`?

Reply:
Not really. It is a rebase not a merge, so it would be bad practice to replace a branch which is public. More practically, it would mean old versions of TiKV (any version without this PR) would no longer build because they would pull the tikv2 branch which is not backwards compatible with the tikv branch due to changes to sysinfo.

Reply:
Why? I thought there was a lock file.

Reply:
GitHub will garbage-collect commits which are not on a branch. It will also cause problems if anyone tries to `cargo update`.